amacc.c

```diff
@@ -1231,6 +1250,23 @@ int *codegen(int *jitmem, int *jitmap)
         case MUL:
             *je++ = 0xe49d1004; *je++ = 0xe0000091; // pop {r1}; mul r0, r1, r0
             break;
+        case DIV:
+        case MOD:
+            *je++ = 0xe52d0004;                     // push {r0}
+            if (elf) {
+                tmp = (int) plt_func_addr[i - OPEN];
+            } else {
+                void *handle = dlopen("libgcc_s.so.1", 1);
+                if (!handle) fatal("libgcc_s.so.1 open error!");
+                tmp = (int) dlsym(handle, scnames[i - OPEN]);
+            }
+            *je++ = 0xe49d0004 | (1 << 12); // pop r1
+            *je++ = 0xe49d0004 | (0 << 12); // pop r0
+            *je++ = 0xe28fe000;                          // add lr, pc, #0
+            if (!imm0) imm0 = je;
+            *il++ = (int) je++ + 1;
+            *iv++ = tmp;
+            break;
```

Comment:
No need to add these lines for DIV and MOD.
`default:` case shall handle all of the system calls.
It obviously you copy the code from `default:`, this should be fixed.

Reply:
The DIV and MOD is outside the `default` case for some reasons:
1. AST generated div and mod same as other operator, not like typical system calls.
2. JIT need to load `libgcc_s` in runtime.

Reply:
Actually, I am confuse what is the function of line 1266 to line 1267?

Reply:
I've tested on my local, `DIV` and `MOD` can be merge into default switch case.
You can leave this optimization in another PR, these lines is okay for me now.

Dear @jserv could you answer the line 1266 and 1267 for `il` and `iv`? 
I'm not familiar with the compiler part.

Reply:
It was a historical kludge while I was to perform "MOV PC, LR" trick: (so-called Traditional ARM instruction)
http://www.toves.org/books/armsub/

That should be replaced by preferred instruction for ARMv7-A, `BX lr`.

Reference:
* https://wiki.ubuntu.com/ARM/Thumb2PortingHowto
* https://lists.denx.de/pipermail/u-boot/2017-March/282897.html

Reply:
I tend to merge this pull request and then leave further refactoring and/or enhancements.